docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: model
+title: Mapping ICD10CM Codes with Corresponding Causes and Claim Analysis Codes
+author: John Snow Labs
+name: icd10cm_cause_claim_mapper
+date: 2023-05-11
+tags: [en, licensed, chunk_mapping, icd10cm, cause, claim]
+task: Chunk Mapping
+language: en
+edition: Healthcare NLP 4.4.0
+spark_version: 3.0
+supported: true
+annotator: ChunkMapperModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pretrained model maps ICD10CM codes, subsequently providing corresponding causes and generating claim analysis codes for each respective ICD10CM code.
```

Comment:
ICD10CM should be ICD-10-CM

Reply:
Add as a new sentence what if there is no claim code of an icd10cm code. (It returns None)

Reply:
Done.